ice.c

```diff
@@ -339,6 +384,18 @@ static gboolean janus_ice_handles_check(gpointer user_data) {
 	}
 	janus_mutex_unlock(&old_handles_mutex);
 
+	if(janus_force_rtcpmux_blackhole_fd > 0) {
```

Comment:
Simply changed the initialization to 0: since we only use it if it's >0, then it should be ok.